graphql_compiler/tests/test_emit_output.py

```diff
@@ -94,74 +92,6 @@ def test_simple_traverse_filter_output(self):
         received_match = emit_match.emit_code_from_ir(compound_match_query, None)
         compare_match(self, expected_match, received_match)
 
-    def test_output_inside_optional_traversal(self):
```

Comment:
I removed this test because of a couple of reasons:
- The test tests functionality that is already covered in `test_compiler.py`.
- Unfortunately, even though it aims to test a narrow bit of functionality (emitting MATCH code), it depends on a vast portion of the compiler: the post-lowering IR, conversion to `MatchQuery` and then `CompoundMatchQuery` object, and, had I kept it, would have also depended on the `QueryMetadataTable`.
- Also unfortunately, the test was not written using the same schema as the other tests, which meant that adding the `QueryMetadataTable` required creating real GraphQL types for this new schema, etc.

All of this put together meant that this was not an effective test anymore, and was not worth the amount of effort it would have taken to turn it into one.